lib/Backend/IRBuilder.cpp

```diff
@@ -4451,6 +4498,7 @@ IRBuilder::BuildProfiledElementCP(Js::OpCode newOpcode, uint32 offset)
         this->DoClosureRegCheck(layout->Instance);
     }
 
+    Js::OpCodeUtil::ConvertNonCallOpToNonProfiled(newOpcode);
```

Comment:
>ConvertNonCallOpToNonProfiled [](start = 20, length = 29)

nit: its kind of nice consistency to call BuildProfiled* methods on profiled opcodes and let them do the conversion to non-profiled.